apps/prairielearn/assets/scripts/lib/histogram.ts

```diff
@@ -0,0 +1,141 @@
+declare global {
+  interface Window {
+    d3: any;
+  }
+}
```

Comment:
Once you get d3 imports working, note that actual types are available via [`@types/d3`](https://www.npmjs.com/package/@types/d3).

Reply:
d3 imports are working and added `@types/d3` in [2294ce6](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/2294ce6fe8294dda929e0be0563c3e2316102eac).